cypress/support/pages/signIn.pageObject.js

```diff
@@ -15,6 +15,10 @@ class SignInPageObject extends PageObject {
     return cy.getByDataCy('sign-in-btn');
```

Comment:
Ensure that the `getByDataCy` command correctly returns the element with the data-cy attribute 'sign-in-btn'.